index.html

```diff
@@ -15,11 +15,54 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+<body class="add-background-color-main">
+
+  <div data-view="entry-form">
+    <form id="entry-form">
+
+      <header class="adjust-nav">
+        <div class="container">Code Journal</div>
+      </header>
+      <main>
+        <h3 class="container adjust-title">New Entry</h3>
+        <div class="container">
```

Comment:
You have three separate containers in this `main` element. You should just put the `container` class on the `main` element so you don't have to keep making new ones.